### DISCUSSION TITLE
Document type editor does not show datatype previews

### DISCUSSION THREAD

**Discussion by kjac** [umbraco/Umbraco-CMS#17178] [Features and ideas]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

V14 RC5

### Bug summary

In the previous versions of Umbraco, the document type editor would show previews of the datatypes for each property. These would be really nice to bring back in V14:

![image](https://github.com/umbraco/Umbraco-CMS/assets/7405322/ff4a27f2-cdc2-43a8-8f94-06015913029a)

### Steps to reproduce

1. Create a doctype with a property.
2. Observe how the gray box representing the property does not display a preview of the property datatype.

**Comment by iOvergaard** [comment]
On the other hand, we do not wish to load the entire property editor in this view, which is the reason that we opted out of showing anything here. I realise it looks a bit "odd" if you were used to anything different. I wonder whether we can put a thumbnail or something in their place. We would really like to avoid loading the likes of TinyMCE and Monaco Editor just to edit a document type :-)

I will convert this to a discussion to see if we can come up with some good ideas for this area!
